test/verify/check-system-info

```diff
@@ -512,6 +512,36 @@ revision\t: 2.3 (pvr 004e 1203)
         b.enter_page("/system")
         b.wait_in_text("#system-usage-cpu-progress + td", "of 2 CPUs")
 
+        # /proc/cpuinfo on s390x (reduced)
+        m.write("/tmp/cpuinfo", """vendor_id       : IBM/S390
+# processors    : 2
```

Comment:
and here I thought the pound symbol was a typo ;)

Reply:
Hehe, no -- I copied this verbatim from the `cat /proc/cpuinfo` output from the bug report.